docs/shared/guides/nx-and-angular-cli.md

```diff
@@ -7,36 +7,54 @@ width="100%" /%}
+
+| Feature/Tool                                                                                               | Angular CLI     | Nx            |
+| ---------------------------------------------------------------------------------------------------------- | --------------- | ------------- |
+| Create Angular Apps                                                                                        | ✅              | ✅            |
+| Generate Angular Components, Services, etc.                                                                | ✅              | ✅            |
+| Building & Bundling                                                                                        | ✅              | ✅            |
+| Local Development Server                                                                                   | ✅              | ✅            |
+| Code Schematics                                                                                            | ✅              | ✅            |
+| Automated Update with Migrations                                                                           | ✅              | ✅ (Enhanced) |
+| Generators                                                                                                 | ✅ (Schematics) | ✅            |
+| Executors                                                                                                  | ✅ (Builders)   | ✅            |
+| Advanced Generators (e.g. Module Federation, Tailwind,...)                                                 | ❌              | ✅            |
+| Integrated Tooling (Jest, Cypress, Playwright etc.)                                                        | ❌              | ✅            |
+| Support for [single-project Workspaces](/getting-started/tutorials/angular-standalone-tutorial)            | ✅              | ✅            |
+| First-Class [Monorepo Support](/getting-started/tutorials/angular-monorepo-tutorial)                       | ❌\*            | ✅            |
+| [Enforced Module Boundaries](/core-features/enforce-module-boundaries)                                     | ❌              | ✅            |
+| Interactive [Project Graph](/core-features/explore-graph)                                                  | ❌              | ✅            |
+| Task Graph                                                                                                 | ❌              | ✅            |
+| [Running Tasks in Parallel](/recipes/running-tasks/run-tasks-in-parallel)                                  | ❌              | ✅            |
+| Building, Testing [Only What is Affected](/core-features/run-tasks#run-tasks-on-projects-affected-by-a-pr) | ❌              | ✅            |
+| [Local Caching](/core-features/cache-task-results)                                                         | ❌              | ✅            |
```

Comment:
To be fair angular has local caching, since a few versions, see: https://angular.io/cli/cache

Reply:
yeah but this is a little different. This is - unless it changed - the webpack level persistent cache that gets written into the `.ng` location to speed up rebuilds. The Nx cache is agnostic and actually leverages that cache underneath as well.

Reply:
@bulldog98 I added some clarification. Thanks